run/types/testing.ts

```diff
@@ -204,12 +208,13 @@ export type AccessibilityId =
   | 'Edit group'
   | 'Group name text field'
   | 'OK'
```

Comment:
I think some of those should be removed, at least OK was replaced by Okay

Reply:
OK is only used in voice calls test and that's currently broken, will remove once I fix that test (or when calls work again)

Reply:
https://optf.atlassian.net/browse/QA-352

Have made a ticket, will look into fixing this test. Not sure how to fully do it as the calls functionality is pretty broken in general.